zkevm-circuits/src/tx_circuit/sign_verify.rs

```diff
@@ -0,0 +1,991 @@
+// Naming notes:
+// - *_be: Big-Endian bytes
```

Comment:
Do we have a convention about using le/be in the code?

If we expect this mixed be/le encodings to repeat a lot in the code, maybe we have to consider to make a BigEndian/LittleEndian types in a closer future to avoid confusions.

Reply:
> Do we have a convention about using le/be in the code?

I don't think so, but we definitely use both.  For example, all Ethereum standards use big endian, but the field type we use uses little endian, so we will encounter conversions around the code. 

> If we expect this mixed be/le encodings to repeat a lot in the code, maybe we have to consider to make a BigEndian/LittleEndian types in a closer future to avoid confusions.

I find this idea interesting but I'm not sure how to do this cleanly.  Because we use different types to represent the bytes: array, slice, vector, writer...  So what I try to do is, whenever I have some bytes that represent an integer, add `_le` or `_be` to annotate the Endianness.

Reply:
Agree, no clear idea about how to do it also, just to keep this in mind :)